app/models/agents/typeform_response_agent.rb

```diff
@@ -31,58 +31,59 @@ class TypeformResponseAgent < Agent
         * `until` - Specify date and time, to limit request to responses submitted until the specified date and time, e.g. `1979-05-27 05:00:00`, `January 5 at 7pm` [more valid formats](https://github.com/mojombo/chronic).
         * `mapping_object` - Specify the mapping definition object where any hidden_variables can be mapped with a single value.
         * `bucketing_object` - Specify the bucketing definition object where any hidden_variables can be broken into a specific bucket.
+
+        If you specify `mapping_object` you must set up something like this:
+
+
+            "city_id": {
+              "1": "London",
+              "2": "New York"
+            },
+            "country_id": {
+              "1": "UK",
+              "2": "US"
+            }
+
+        If you specify `bucketing_object` you must set up something like this:
+
+            "tc": {
+              "1": "First Trip",
+              "2-10": "2 - 10",
+              "11-20": "11 - 20",
+              "21-50": "21 - 50",
+              "51-100": "51 - 100",
+              "100+": "101 and more"
+            }
     MD
 
     event_description <<-MD
       Events look like this:
-        {
+
           "score": 9,
           "comment": "Love the concept and the food! Just a little too expensive.",
           "id": "62e3caeaca5100adf84f61708ad69960",
           "created_at": "2017-08-12 19:16:02",
           "answers": {
-            {
-               "field": {
-                 "id": "58048684",
-                 "type": "opinion_scale"
-               },
-               "type": "number",
-               "number": 3
-             },
-             {
-               "field": {
-                 "id": "58049393",
-                 "type": "multiple_choice"
-               },
-               "type": "choice",
-               "choice": {
-                 "label": "55 to 64"
-               }
-             },
-             {
-               "field": {
-                 "id": "58049487",
-                 "type": "dropdown"
-               },
-               "type": "text",
-               "text": "South East"
-             },
-             {
-               "field": {
-                 "id": "58049969",
-                 "type": "long_text"
-               },
-               "type": "text",
-               "text": "It is really helpful to see your credit score at any time and if it changes then why gas it changed"
-             }
+            "opinion_scale_58048690": 3,
+            "long_text_58049969": "It is really helpful to see your credit score at any time and if it changes then why gas it changed",
+            "opinion_scale_58048493": 5,
+            "opinion_scale_58048704": 2,
+            "dropdown_58049487": "South East",
+            "opinion_scale_58048049": 10,
+            "opinion_scale_58048684": 3,
+            "opinion_scale_58048687": 3,
+            "multiple_choice_58049393": {
+              "label": "55 to 64"
```

Comment:
Why does this one have a different format?

Reply:
Each tuple is an answer, the key is the combination between field.type and value (text, number, choice..)